### DISCUSSION TITLE
Version history per bullet / page

### DISCUSSION THREAD

**Discussion by Limezy** [athensresearch/athens#926] [Feature Requests]
Definitely need this :) 
And maybe, for each bullet having a way to check date of creation / modification
Who knows, in the future, for each bullet having a way to check who wrote it / modified it

**Comment by jefftangx** [comment]
I think we will get this for free once we use datahike. Might not be worth implementing while we we use datascript, but not sure when datahike will be ready for us, and when we migrate.
